### ISSUE TITLE
drag and drop using some library between lists.

### ISSUE THREAD

**Issue by Himanshu-Singh-Chauhan** [Himanshu-Singh-Chauhan/extension#6]
## UPDATE
https://seniyas.medium.com/react-drag-and-drop-libraries-what-should-we-choose-459ebd79ce01

chose two libs:
- primary library: dnd-kit (actively maintained)
dnd kit can be combined with https://auto-animate.formkit.com/ and framer motion for animations
- framer motion also has dnd, check that out also.
- secondary library: react-beautiful-dnd (no maintainence, only security updates) (we use it when dnd-kit fails to provide what we need)
edit : take a look at use-gesture for secondary library when time comes to add secondary library.

---

other libraries researched:
- **hello-pangea/dnd,** (fork of react-beautiful-dnd) - it is being actively maintained and used by people.
- https://use-gesture.netlify.app/ - really powerful, but gotta write the behaviors ourself.
- react draggable (could be useful)
- https://shopify.github.io/draggable/ - (still maintained by new maintainers)
- ig one more was there, it was really good, but dang forgot the name, def its somewhere in my discord or telegram.

libraries not to touch:
- react-dnd (not that useful) (old) (rough api) (kinda dead) (useful for file or something else drag and drop from outer application)
- react-beautiful-dnd (its dead now, not maintained, last release was in 2020)
- pragmatic dnd (new lib by atlassian)
- basically avoid atlasssian products

---

### TASK
compare libs and arrive at one, with most features, less setup and overhead, active library not dead
